app/chat/build.gradle

```diff
@@ -24,10 +24,10 @@ repositories {
 }
 
 dependencies {
-    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
-    compile "io.ktor:ktor-server-netty:$ktor_version"
-    compile "io.ktor:ktor-websockets:$ktor_version"
-    compile "ch.qos.logback:logback-classic:$logback_version"
-    testCompile "io.ktor:ktor-server-test-host:$ktor_version"
+    api "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
```

Comment:
Can we use `implementation` instead of `api`?

Reply:
it doesn't matter actually: we're not publishing. But let's change it to api->implementation everywhere